tests/lib/pkgdb.sh

```diff
@@ -438,6 +438,13 @@ distro_install_build_snapd(){
         if [[ "$SPREAD_SYSTEM" = ubuntu-14.04-* ]] && [ "$SPREAD_REBOOT" = 0 ]; then
             REBOOT
         fi
+    elif [ -n "$PPA_VALIDATION_NAME" ]; then
+        apt install -y snapd
+        add-apt-repository -y "$PPA_VALIDATION_NAME"
+        apt update
+        apt install -y --only-upgrade snapd
+        add-apt-repository --remove "$PPA_VALIDATION_NAME"
+        apt update
```

Comment:
This is not strictly needed, apt will use its sources.list{,.d}. But its still fine to have it here.

Reply:
Nice, Thanks for the review!!